src/views/Signup/SignUp.scss

```diff
@@ -99,6 +99,12 @@ html {
   text-align: center;
 }
 
+.headsi {
```

Comment:
`.headsi` as a className doesn't make sense to me, please change this to something that make sense?